### DISCUSSION TITLE
How to create multiple dynamic routes at root

### DISCUSSION THREAD

**Discussion by CaptainT33mo** [vercel/next.js#11908] [Help] [answered]
Hi, 
I am working on a site where we need to have dynamic URLs as follows:
`pages/[username]`
`pages/[jobCategory]/[jobTitle]`
`pages/[groupName]/[postId]`

How to create these pages as dynamic routes at the root. I've tried creating multiple dynamic routes but it gives error as `you cannot use different slug names for the same dynamic path`.

I am new to Next.JS and any help would be greatly appreciated.

**Comment by emilforsmann** [comment, marked as answer]
To my knowledge that's not possible. 
How should nextjs know which dynamic value you're parsing belongs to what. 
what you can do is put them in folders:
- pages/u/[username]
- pages/j/[jobCategory]/[jobTitle]
- pages/g/[groupName]/[postId]
